email_test.go

```diff
@@ -62,7 +65,7 @@ func TestEmail_Send(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	expBody := "From: from@example.com\nTo: to@example.com\nSubject: subj\nContent-Transfer-Encoding: quoted-printable\nMIME-version: 1.0\nContent-Type: text/html; charset=\"UTF-8\"\nDate: Thu, 10 Feb 2022 23:33:58 +0000\n\nsome text\r\n"
+	expBody := "From: from@example.com\nTo: to@example.com\nSubject: subj\nMIME-version: 1.0\nDate: Thu, 10 Feb 2022 23:33:58 +0000\nContent-Transfer-Encoding: quoted-printable\nContent-Type: text/html; charset=\"UTF-8\"\n\nsome text\r\n\r\n"
```

Comment:
we still don't know why extra \r\n needed here?

Reply:
Still in progress.

Reply:
Done.